pkg/output/output.go

```diff
@@ -130,6 +131,13 @@ func (iwe *InternalWrappedEvent) SetOperatorResult(operatorResult *operators.Res
 	iwe.OperatorsResult = operatorResult
 }
 
+type RequestResponse struct {
```

Comment:
I’d rather name this `Steps` (`steps` in JSON) or something else that’s shorter and more direct, just to avoid confusion. I mean, what does `request_response` even represent - request or response from what?

Also, write some inline docs for it.